ddocs/medic/views/doc_summaries_by_id/map.js

```diff
@@ -42,9 +44,6 @@ function(doc) {
     if (reference) {
       subject.value = reference;
       subject.type = 'reference';
-    } else if (doc.fields && doc.fields.place_id) {
-      subject.value = doc.fields.place_id;
-      subject.type = 'id';
```

Comment:
Nice simplification! It's weird that this was differentiated in the first place...

Reply:
I think it was differentiated because we only expected uuids to be in `place_id` fields. I think this was added with 
Wealth Quintiles? It was avoiding querying contact_by_reference for things we knew could only be uuids (because places had no shortcodes at the time).